package.json

```
@@ -19,8 +19,15 @@
     "dev:backend:realtime": "turbo dev --log-order stream --filter '@apps/hash-realtime' --",
     "dev:backend:search-loader": "turbo dev --log-order stream --filter '@apps/hash-search-loader' --",
     "dev:frontend": "turbo dev --log-order stream --filter '@apps/hash-frontend' --",
-    "dev:graph": "yarn workspace @apps/hash-graph dev",
-    "dev:graph:test-server": "yarn workspace @apps/hash-graph dev:test-server",
+    "start": "turbo run start start:healthcheck --filter @apps/hash-graph --filter @rust/type-fetcher --filter @apps/hash-api --filter @apps/hash-ai-worker-ts --filter @apps/hash-integration-worker --filter @apps/hash-realtime --filter @apps/hash-frontend --env-mode=loose --ui tui --continue",
```

We're not using `realtime` so I suggest we take the opportunity to remove from here. Can restore it when we do again
```suggestion
    "start": "turbo run start start:healthcheck --filter @apps/hash-graph --filter @rust/type-fetcher --filter @apps/hash-api --filter @apps/hash-ai-worker-ts --filter @apps/hash-integration-worker --filter @apps/hash-frontend --env-mode=loose --ui tui --continue",
```

---

Good idea, I removed the search-loader as well so we don't need to build dependencies for an `echo` command.